src/app/models.py

```diff
@@ -28,6 +28,7 @@ class LicenseNames(models.Model):
     name = models.CharField(max_length=200)
 
 class LicenseRequest(models.Model):
+    licenseAuthorName = models.CharField(max_length=100)
```

Comment:
Please add a default of "" (blank)

Reply:
@goneall I have updated the PR.
```suggestion
    licenseAuthorName = models.CharField(max_length=100, default="")
```